code/script.js

```diff
@@ -1,51 +1,196 @@
 // Variables that point to selected DOM elements
-const chat = document.getElementById('chat');
+const chat = document.getElementById("chat");
+const inputWrapper = document.getElementById("input-wrapper");
+const form = document.getElementById("name-form");
+const inputValue = document.getElementById("name-input");
 
 // If you need any global variables that you can use across different functions, declare them here:
-
+let currentQuestion = 0;
+let userName = "";
+let favorite = "";
+const setTimeOutTime = 1200;
 
 // Declare your functions after this comment
 
 // This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
   // the if statement checks if the sender is 'user' and if that's the case it inserts an html senction inside the chat with the posted message
-  if (sender === 'user') {
+  if (sender === "user") {
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/user-avatar.png" alt="User" />  
       </section>
-    `
+    `;
     // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
-  } else if (sender === 'bot') {
+  } else if (sender === "bot") {
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/catbot.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
-    `
+    `;
   }
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight;
-}
+};
 
 // Starts here
 const greetUser = () => {
   // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
-}
+  showMessage(
+    "Welcome to Cat Bot the Chat bot. What's your first name?",
+    "bot"
+  );
+  chat.scrollTop = chat.scrollHeight;
+};
+
+const favoritePersonality = () => {
+  showMessage(
+    `Hello ${userName}! Which cat personality is your favorite?`,
```

Comment:
Love the idea and styling of your page! Over all a great job 👍 You seem to be following the general requirements and more and your code is simple to read. The only thing missing for me is some comments to make it more clear wich function does what, but I think it is quite self explanatory anyways, so not necessarily anything you have to change.

Reply:
Thank you! Good suggestion with comments on the code explaining the functions.